### ISSUE TITLE
File names in radiance folder

### ISSUE THREAD

**Issue by mostaphaRoudsari** [ladybug-tools/honeybee-grasshopper-radiance#20]
Hi @chriswmackey, the folder looks really nice. Am I correct that all the opaque geometry and all the aperture will always be written to a single file? In that case are you fine if we do not include the model name in the name and keep it as aperture.rad and envelope or opaque.rad?

**Comment by chriswmackey** [comment]
Yes, they all get written into one file and I am ok with using those names. I'm moving this issue to honeybee-radiance since it's a core library thing and doesn't have anything to do with the grasshopper interface.
